doc/model/train-energy.md

```diff
@@ -0,0 +1,60 @@
+# Fit energy
+
+## Fitting network
+
+The `fitting_net` section tells DP which fitting net to use.
+
+The json of `dipole` type should be provided like
+
+```json
+	"fitting_net" : {
+		"type": "dipole",
+		"sel_type": [0],
+		"neuron": [100,100,100],
+		"resnet_dt": true,
+		"seed": 1,
+	},
+```
+
+The json of `polar` type should be provided like
+
+```json
+	"fitting_net" : {
+	   	"type": "polar",
+		"sel_type": [0],
+		"neuron": [100,100,100],
+		"resnet_dt": true,
+		"seed": 1,
+	},
+```
```

Comment:
They are fitting nets for tonsorial properties

Reply:
Fixed.